tests/pyoozie/test_interactive.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from __future__ import print_function
+
+import datetime
+import os
+
+import pyoozie
+import pytest
+import pywebhdfs.webhdfs
+import six
+
+
+@pytest.mark.skipif(not bool(os.environ.get(str('INTERACTIVE'))), reason='Requires INTERACTIVE=1 env var')
```

Comment:
This test isn't run unless this env var is present because Travis CI doesn't start up an Oozie server yet (and this test is meant to be interactive, i.e. it will pause for a response before cleaning up its coordinator created on Oozie).